verification/curator-service/ui/package.json

```diff
@@ -10,7 +11,10 @@
     "@types/node": "^12.12.37",
     "@types/react": "^16.9.34",
     "@types/react-dom": "^16.9.6",
+    "axios": "^0.19.2",
+    "bootstrap": "^4.4.1",
     "react": "^16.13.1",
+    "react-bootstrap": "^1.0.1",
```

Comment:
This is a bold move ;-) Let's see what our designers are going to make us build, I'd have suspected react-material or something alike.

https://material-ui.com/

Reply:
Ha, yeah -- I imagine this will change. :-)